jobbergate-agent/jobbergate_agent/jobbergate/schemas.py

```diff
@@ -78,3 +78,45 @@ class SlurmJobData(pydantic.BaseModel, extra="ignore"):
     job_state: Optional[str] = None
     job_info: Optional[str] = None
     state_reason: Optional[str] = None
+
+
+class InfluxDBMeasurement(TypedDict):
+    """
+    Map each entry in the list returned by `InfluxDBClient(...).get_list_measurements(...)`.
+    """
+
+    name: INFLUXDB_MEASUREMENT
+
+
+class InfluxDBMeasure(TypedDict):
```

Comment:
Why a `TypedDict` instead of a `dataclass`?

Reply:
there's no specific why, I just thought in `TypedDict` and used it :)